packages/openstad-event-browser-widgets/src/components/events/event-map.tsx

```diff
@@ -18,7 +23,7 @@ export function EventMap({ events }: any) {
             ]}
           >
             <Popup>
-              <Link href={`/${event.id}`}>{event.name}</Link>
+              <Link to={`${prefixUrl}${slug}/${event.id}`}>{event.name}</Link>
```

Comment:
I don't think `to` is a prop that is available in Wouter's `<Link>` component: https://github.com/molefrog/wouter/tree/v2.7.4#link-hrefpath-

Reply:
> Could you check if replacing `to` props with `href` prop still works?

Changed 'to' to href, seems to work